tutorcodejail/tests/test_patches.py

```diff
@@ -0,0 +1,10 @@
+"""Tests to verify number of patches"""
+
+
+from tutorcodejail.plugin import patches
+
+
+def test_patches():
+    all_patches = patches()
+
+    assert len(all_patches) == 5
```

Comment:
Please, fix this test:
1. The module docstring must describe what the module does not the test implemented in it
2. Use the unittest Python library
3. Add docstrings for each test

Reply:
I already fixed it